milliner/README.md

```diff
@@ -13,7 +13,7 @@ additional settings, volumes, ports, etc.
 | Environment Variable | Confd Key            | Default            | Description                                                                                       |
 | :------------------- | :------------------- | :----------------- | :------------------------------------------------------------------------------------------------ |
 | MILLINER_DRUPAL_URL  | /milliner/drupal/url | drupal:80          | Drupal URL                                                                                        |
-| MILLINER_FCREPO_URL  | /milliner/fcrepo/url | fcrepo/fcrepo/rest | Fcrepo Rest API URL                                                                               |
+| MILLINER_FCREPO_URL  | /milliner/fcrepo/url | fcrepo:8080/fcrepo/rest | Fcrepo Rest API URL                                                                          |
```

Comment:
Just noticed now, but the URL variables for the microservices in the README.md files are missing the prefixed `http://`, so this should probably be fixed across all microservices.

Reply:
Also, can you make sure the alignment of the tables are kept? This can be done with <https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one> if using VS Code, and I'm sure other auto-formatters exist for other IDEs.

Reply:
Roger that! Will do.

Reply:
Not sure if I did this right in terms of alignment. I do use that auto-formatter but it seemed a bit thick on moving things around properly. Or perhaps it's operator was? 😄

Reply:
Also tested this on my local. no issues discovered